src/listeners/messageCreate.ts

```diff
@@ -67,13 +67,44 @@ client.on("messageCreate", async message => {
 
 	// Delete non-stringURL messages in review-strings
 	const stringURLRegex = /(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi
+
 	if (message.channel instanceof TextChannel && message.channel.name.endsWith("-review-strings")) {
-		if (!/(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi.test(message.content))
-			await message.delete()
-		else {
-			await message.react("vote_yes:839262196797669427")
-			await message.react("vote_maybe:839262179416211477")
-			await message.react("vote_no:839262184882044931")
+		if (stringURLRegex.test(message.content)) {
+			const urlsWithComments: string[] = [],
+				contentSplit = message.content.split(" "),
+				areUrls = contentSplit.map(w => stringURLRegex.test(w))
+
+			let commentAtStart = "",
+				encounteredUrl = areUrls[0]
+
+			for (let i = 0; i < areUrls.length; i++) {
```

Comment:
You should use for of here instead

Reply:
No, because i need to access multiple arrays with the same index.

Reply:
Right ok

Reply:
But to be completely honest, I dont need areUrls here, I'm going to change this.